### PR TITLE
Form helper doesn't escape value attribute for values entities

### PR DESCRIPTION
**Pull request by davydovanton**
Closes https://github.com/hanami/hanami/issues/709

### CODE REVIEW

File: lib/hanami/helpers/html_helper/empty_html_node.rb [hunk 1]
```diff
@@ -124,7 +124,11 @@ def boolean_attribute(attribute_name, _value)
         end
 
         def attribute(attribute_name, value)
-          %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
+          if attribute_name.to_sym == :value
+            %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{Hanami::Utils::Escape.html(value)}")
+          else
+            %(#{ATTRIBUTES_SEPARATOR}#{attribute_name}="#{value}")
```
Comment (AlfonsoUceda) [inline lib/hanami/helpers/html_helper/empty_html_node.rb]: Shouldn't we escape the value in all cases?

Comment (jodosha) [inline lib/hanami/helpers/html_helper/empty_html_node.rb]: Probably yes.

Please also consider to use `html_attribute` instead https://github.com/hanami/utils/blob/ca91c336ffebb43f1632ffd398b4281b3b7315a5/lib/hanami/utils/escape.rb#L453

Comment (davydovanton) [inline lib/hanami/helpers/html_helper/empty_html_node.rb]: We have some attributes like a `script`. If we escape this attributes we will have a error in layout. What do you think what we should do with this attributes?

Comment (TiteiKo) [inline lib/hanami/helpers/html_helper/empty_html_node.rb]: Could we simply make a list of the attributes that need escaping? (like `value`, `title`, ...)

Comment (jodosha) [inline lib/hanami/helpers/html_helper/empty_html_node.rb]: @TiteiKo We can't predict where developers are going to interpolate user's input.

Comment (jodosha) [inline lib/hanami/helpers/html_helper/empty_html_node.rb]: Alternatively we can make this change in `_value`: https://github.com/hanami/helpers/blob/8f10dbd7475bd28f6197bd667540577cccad1504/lib/hanami/helpers/form_helper/form_builder.rb#L984

WDYT?

Comment (davydovanton) [inline lib/hanami/helpers/html_helper/empty_html_node.rb]: @jodosha good idea 👍

**Comment by davydovanton** [comment]
Okay, I updated `#_attributes` form helper because `#_value` sets default value and we need to escape all values. Also I used `#html_attribute` method instead `#html`.

**Comment by jodosha** [comment]
@davydovanton I don't understand the difference. Why it can't be in `_value`?

**Comment by davydovanton** [comment]
I fixed CI build. 

`_value` used only for the default value. If you have a value attribute in your form, you will redefine this value. That's why I escape value after the merge.

**Comment by davydovanton** [comment]
Also, I checked this in my local app, and all worked correctly 🎉

**Comment by davydovanton** [comment]
@jodosha thanks for help and review 👍

**Pull request merged**